commands/version/lib/git-add.js

```diff
@@ -1,22 +1,56 @@
 "use strict";
 
+const fs = require("fs");
 const log = require("npmlog");
 const path = require("path");
 const slash = require("slash");
+const { workspaceRoot } = require("@nrwl/devkit");
 const childProcess = require("@lerna/child-process");
 
 module.exports.gitAdd = gitAdd;
 
+function maybeFormatFile(filePath) {
+  let prettier;
+  try {
+    // If the workspace has prettier installed, apply it to the updated files
+    const prettierPath = path.join(workspaceRoot, "node_modules", "prettier");
+    // eslint-disable-next-line import/no-dynamic-require, global-require
+    prettier = require(prettierPath);
+  } catch {
```

Comment:
This block could be done only once outside of this function, since it only looks in the workspace root.